.ci/install_cni_plugins.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+#
+# Copyright (c) 2017-2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -e
+
+echo "Retrieve CNI plugins repository"
+go get -d github.com/containernetworking/plugins || true
+pushd $GOPATH/src/github.com/containernetworking/plugins
+
+echo "Build CNI plugins"
+./build.sh
+
+echo "Install CNI binaries"
+cni_bin_path="/opt/cni"
+sudo mkdir -p ${cni_bin_path}
+sudo cp -a bin ${cni_bin_path}
+
+echo "Configure CNI"
+cni_net_config_path="/etc/cni/net.d"
+sudo mkdir -p ${cni_net_config_path}
+
+sudo sh -c 'cat >/etc/cni/net.d/10-mynet.conf <<-EOF
+{
+	"cniVersion": "0.3.0",
```

Comment:
Nit: this is hard-coded twice - might be best to define as a variable so that it can be updated in one place when needed.